src/en/en.json

```diff
@@ -3,7 +3,8 @@
   "links": {
     "home": "/en",
 
-    "installation": "/en/installation",
+    "getStartedDesign": "/en/get-started/design",
+    "getStartedDevelop": "/en/get-started/develop",
```

Comment:
Not directly related to the work done in this PR but it looks like each individual framework installation page is missing from the links data.